bagisto/bagisto
# Fixed #8623

## Issue Reference
#8623 
## Description
Fixed #8623  => Change @lang to trans for copyright footer message, & modified relevant lang files of shop, & Tested it working fine

## How To Test This?
Go to 'Shop'
Click the sign-up page, login page or Forgot Password Page
Scroll down to the footer of the above pages.
Observe that the copyright year is displaying now Current year.
